### PR TITLE
chore: jsonfmt machinelearning

### PR DESCRIPTION
**Pull request by nschonni**
Ran `jsonfmt -w "specification/machinelearning/**/*.json"`

### CODE REVIEW

**Comment by nschonni** [comment]
Fixed the casing issues with model validation is separate commits. They can be removed if prefered. There is an outstanding error still, but the 200 status description says "Internal use only." so maybe that is by-design.
Not sure about the BreakingChange error

File: specification/machinelearning/resource-manager/Microsoft.MachineLearning/stable/2017-01-01/examples/getWebService.json [hunk 1]
```diff
@@ -1,194 +1,193 @@
...
+                  "workclass": {
+                    "type": "String",
+                    "x-ms-isnullable": false
+                  }
                 }
+              }
             }
+          },
+          "package": {
+            "nodes": {
```
Comment (sergey-shandar) [inline specification/machinelearning/resource-manager/Microsoft.MachineLearning/stable/2017-01-01/examples/getWebService.json]: This PR contains changes that shouldn't be included as json formatting.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

**Comment by sergey-shandar** [comment]
@nschonni could you create two separate PRs?

**Comment by nschonni** [comment]
@sergey-shandar if you want to re-open this one, i'll rebase out the non-formatting changes and move them to a second PR

**Comment by nschonni** [comment]
@sergey-shandar I opened up the format-only PR in #6272. I'll submit something with the other commits after that lands

**Pull request closed without merging**